.github/workflows/ci-core.yml

```diff
@@ -284,11 +284,234 @@ jobs:
         env:
           OUTPUT_FILE: ./output.txt
           CL_DATABASE_URL: ${{ env.DB_URL }}
+          USE_FLAKEGUARD: true 
         run: |
           # See: https://github.com/golang/go/issues/69179
           GODEBUG=goindex=0
           ./tools/bin/${{ matrix.type.cmd }} ./...
 
+      - name: Install flakeguard
+        if: ${{ always() && (matrix.type.cmd == 'go_core_tests' || matrix.type.cmd == 'go_core_ccip_deployment_tests') }}
+        shell: bash
+        run: go install github.com/smartcontractkit/chainlink-testing-framework/tools/flakeguard@d71989ddf270d247e61e1c7e23fa65fcc141c73b # flakguard@0.1.0
```

Comment:
This is already installed in the shell script - does it need to be installed again?